server/tournaments/index.ts

```diff
@@ -1701,7 +1748,7 @@ const chatCommands: ChatCommands = {
 			}
 		} else {
 			const tournament = (room.game && room.game.gameid === 'tournament') ? room.game as Tournament : null;
-			if (!tournament) {
+			if (!tournament && cmd !== 'export') {
```

Comment:
just put the command code above here, look at how ``create`` is implemented.

Reply:
Ok, I wasn't sure abt that, the other actions seems more moderator-y. Thanks.